integration_tests/integration/bookedtoday/arrivals/autoMatchingRecords/isSingleMatch.spec.ts

```diff
@@ -113,4 +113,29 @@ context('Is Single Match', () => {
     const searchPage = Page.verifyOnPage(SearchForExistingPage)
     searchPage.name.value().should('contain.text', 'Bob Smith')
   })
+
+  it('Back link navigation is not displayed on confirmation page', () => {
+    cy.task('stubExpectedArrival', expectedArrival)
+    cy.signIn()
+
+    const singleMatchingRecordFoundPage = ChoosePrisonerPage.selectPrisoner(
+      expectedArrival.id,
+      SingleMatchingRecordFoundPage
+    )
+    singleMatchingRecordFoundPage.backNavigationLink().should('exist')
+    singleMatchingRecordFoundPage.continue().click()
+
+    const imprisonmentStatusPage = Page.verifyOnPage(ImprisonmentStatusPage)
+    imprisonmentStatusPage.backNavigationLink().should('exist')
+    imprisonmentStatusPage.imprisonmentStatusRadioButton('on-remand').click()
+    imprisonmentStatusPage.continue().click()
+
+    const checkAnswersPage = Page.verifyOnPage(CheckAnswersPage)
+    checkAnswersPage.backNavigationLink().should('exist')
+    cy.task('stubCreateOffenderRecordAndBooking', { arrivalId: expectedArrival.id })
+    checkAnswersPage.addToRoll().click()
```

Comment:
could maybe test back link here and go back a couple pages then click through to check back not being present